test/ListTest.php

```diff
@@ -14,13 +14,7 @@ public function testList()
 </ul>
 EOT;
 
-        $expected =<<<'EOT'
- 	* Item 1
- 	* Item 2
- 	* Item 3
-
-
-EOT;
+        $expected = "    * Item 1\n    * Item 2\n    * Item 3\n";
```

Comment:
Why did you change the formatting here, please use heredoc/nowdoc to demonstrate the change